recipes/poco/all/conanfile.py

```diff
@@ -21,10 +21,12 @@ class PocoConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "enable_fork": [True, False],
```

Comment:
`fork` is not available on Windows.
So I think it should be conditionally removed in `config_options`.